src/js/themes/base.js

```diff
@@ -305,13 +305,28 @@ export const generate = (baseSpacing = 24, scale = 6) => { // 24
         radius: '4px',
         width: '2px',
       },
+      box: {
```

Comment:
I think `box` is too vague here.

Reply:
But, I think our naming should be more attuned to the person creating the theme than to our own internal names.

I also think we should allow for people to theme the CheckBox to be a solid color box when selected.

Here's a suggestion for `theme.checkBox`:
```
checkBox: {
  box: {
    background:
    border: {
      color: {
        dark:
        light:
      }
      thickness:
    },
    round:
  },
  check: {
    color: {
      dark:
      light:
    }
    width: '4px',
  },
  checked: {
    box: {
      background:
      border: {
        color: {
          dark:
          light:
        }
        thickness:
      },
      round:
    }
  },
```

Reply:
I agree this works well. but we still need one for knob i believe. to customize things in the toggle mode.